pkg/lib/form-layout.less

```diff
@@ -43,8 +47,17 @@
 // a <form class="ct-form-layout"> and layout magic happens.
 
 .ct-form-layout {
+  // Locally redefine padding back to Bootstrap values
+  @padding-large-vertical: 10px;
+  @padding-small-vertical: 5px;
+  @padding-small-horizontal: 10px;
+  // Bootstrap & PatterFly use a 1px border around widgets
+  @border-width: 1px;
+  // Computed widget height based on font-size * line height + extra
+  @widget-height: @line-height-computed + (@padding-base-vertical * 2) + (@border-width * 2);
```

Comment:
The comment and especially the `@widget-height` computation should answer @martinpitt's question "why 26px?" (paraphrased)

Reply:
So, line-height-computed is a variable from bootstrap or patternfly variables.
variables.less is stable that we can reuse these values just like this?

Reply:
Yep! `@line-height-computed` hasn't changed in so many years. And that's basically what we're building on top of anyway — so even if it were to change, we want that change reflected here too.

I could've broken it down further to do `(font-size * line-height)` in the variable too, but there's already a computed version. Do you think I should expand it to `(@font-size-base * @line-height-base) + (@padding-base-vertical * 2) + (@border-width * 2)` here or is computed enough?

Reply:
It's ok as is. Thanks. Just wanted to make sure this is a variable that we can rely on.